workflow-templates/python-deploy-artifacts.yml

```diff
@@ -35,7 +35,6 @@ jobs:
           path: python/dist/
 
       - name: Publish distribution to PyPI
-        uses: pypa/gh-action-pypi-publish@master
+        uses: pypa/gh-action-pypi-publish@release/v1
         with:
-          password: ${{ secrets.PYPI_API_TOKEN }}
           packages_dir: python/dist/
```

Comment:
```suggestion
```

Looks like providing the distribution directory location is no longer needed as long as it is a `dist` directory.

https://github.com/marketplace/actions/pypi-publish#customizing-target-package-dists-directory
Customizing target package dists directory
You can change the default target directory of dist/ to any directory of your liking. The action invocation would now look like:

- name: Publish package to PyPI
  uses: pypa/gh-action-pypi-publish@release/v1
  with:
    packages-dir: custom-dir/

Reply:
And we've not been providing it with our other recent secure uploads for PyPi deployment.

Reply:
So just removing that line should be fine?